### PR TITLE
update docs about submit event in zh-CN

### PR DESCRIPTION
**Pull request by awxiaoxian2020**

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
Hi, @yin1999 
In a pull request, I originally only submit a file changes, but due to the **fuzzy match mechanism**  of the conversion tool, I submitted two file changes. The two changes are related maintenance difficulty is not particularly big. But when we meet there is no connection between them with the similar name, fuzzy matching is reasonable? I mean, this will increase maintenance difficulty.

**Comment by yin1999** [comment]
> Hi, @yin1999 In a pull request, I originally only submit a file changes, but due to the **fuzzy match mechanism** of the conversion tool, I submitted two file changes. The two changes are related maintenance difficulty is not particularly big. But when we meet there is no connection between them with the similar name, fuzzy matching is reasonable? I mean, this will increase maintenance difficulty.

This cli is used for batch conversion on a folder, not for a specific document.

**Comment by awxiaoxian2020** [comment]
> 

I know about it. I mean it will affect more folders more than one as we see.

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 1]
```diff
@@ -1,64 +1,88 @@
...
-   <td><code>cancelable</code> {{readonlyInline}}</td>
-   <td>{{jsxref("Boolean")}}</td>
-   <td>Whether the event is cancellable or not.</td>
-  </tr>
- </tbody>
+{{APIRef}}
+
+当表单 {{HtmlElement("form")}} 提交的时候触发 `submit` 事件。
+
+<table class="属性">
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:18]: `properties` is a specific css style for `属性表`, see <https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN#%E5%B1%9E%E6%80%A7%E8%A1%A8>:

```suggestion
<table class="properties">
```

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 2]
```diff
@@ -1,64 +1,88 @@
...
+  <tbody>
+    <tr>
+      <th>冒泡</th>
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:21]: 是否冒泡

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 3]
```diff
@@ -1,64 +1,88 @@
...
+      <td>能（虽然其通常被看作不冒泡的简单事件）</td>
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:22]: “能”--->“是”

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 4]
```diff
@@ -1,64 +1,88 @@
...
+    </tr>
+    <tr>
+      <th>可撤销的</th>
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:25]: 是否可取消

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 5]
```diff
@@ -1,64 +1,88 @@
...
+      <td>是</td>
+    </tr>
+    <tr>
+      <th>接口</th>
+      <td>{{DOMxRef("SubmitEvent")}}</td>
+    </tr>
+    <tr>
+      <th>事件处理属性</th>
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:33]: 事件处理器属性

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 6]
```diff
@@ -1,64 +1,88 @@
...
+      <td>{{domxref("GlobalEventHandlers.onsubmit")}}</td>
+    </tr>
+  </tbody>
 </table>
+
+注意 `submit` 事件只能作用于 `<form>` 元素本身，不能作用于 {{HtmlElement("button")}} 或者 {{HtmlElement('input/submit', '&lt;input type="submit"&gt;')}}。但 {{domxref("SubmitEvent")}} 被用于表单提交时，会携带 {{domxref("SubmitEvent.submitter", "submitter")}} 属性，这是触发提交请求时会调用的按钮。
+
+当用户点击提交按钮（{{HtmlElement("button")}} 或 {{HtmlElement('input/submit', '&lt;input type="submit"&gt;')}}），亦或是在表单里输入时（e.g. {{HtmlElement('input/text', '&lt;input type="text"&gt;')}}）按下 <kbd>Enter</kbd> 键，`submit` 事件将会被触发。直接调用 {{domxref("HTMLFormElement.submit()", "form.submit()")}} 方法时，事件不会发送到表单。
+
+> **备注：** 当试图提交一个未通过 [验证](/zh-CN/docs/Learn/Forms/Form_validation)的表单时，会触发 {{domxref("HTMLInputElement/invalid_event", "invalid")}} 事件。在这种情况下，表单提交将会被验证机制阻止。因此，`sumbit` 事件也就不存在了。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:43]:
```suggestion
> **备注：** 当试图提交一个未通过[验证](/zh-CN/docs/Learn/Forms/Form_validation)的表单时，会触发 {{domxref("HTMLInputElement/invalid_event", "invalid")}} 事件。在这种情况下，表单提交将会被验证机制阻止。因此，`sumbit` 事件也就不存在了。
```

File: files/zh-cn/web/api/htmlformelement/submit_event/index.md [hunk 7]
```diff
@@ -1,64 +1,88 @@
...
+}
+
+const form = document.getElementById('form');
+const log = document.getElementById('log');
+form.addEventListener('submit', logSubmit);
+```
+
+### 结果
+
+{{EmbedLiveSample("Examples")}}
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit_event/index.md:75]: 请匹配ID，在简体中文翻译指南中对此有所说明：

```suggestion
{{EmbedLiveSample("示例")}}
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by yin1999** [comment]
> I know about it. I mean it will affect more folders more than one as we see.

Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:

https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172

**Comment by yin1999** [comment]
> Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:
> 
> https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172

If you want to match folder (not a prefix), you could run command like this: `yarn md h2m web/api/htmlformelement/submit/ --locale zh-CN --mode dry` (adding a slash in the end)

**Comment by awxiaoxian2020** [comment]
> > I know about it. I mean it will affect more folders more than one as we see.
> 
> Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:
> 
> https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172

I think we need a discussion for it, I will start it when I'm available.

**Comment by awxiaoxian2020** [comment]
> > Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:
> > https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172
> 
> If you want to match folder (not a prefix), you could run command like this: `yarn md h2m web/api/htmlformelement/submit/ --locale zh-CN --mode dry` (adding a slash in the end)

OK!Thanks!

**Comment by yin1999** [comment]
> > > Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:
> > > https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172
> > 
> > 
> > If you want to match folder (not a prefix), you could run command like this: `yarn md h2m web/api/htmlformelement/submit/ --locale zh-CN --mode dry` (adding a slash in the end)
> 
> OK!Thanks!

This seems not work, the slug will not end with `/` and the cli **should** match folder (not prefix), I'll working on it.

File: files/zh-cn/web/api/htmlformelement/submit/index.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+  - 表单元素
+translation_of: Web/API/HTMLFormElement/submit
+---
+{{ ApiRef("HTML DOM") }}
+
+**`HTMLFormElement.submit()`** 用来提交表单 {{HtmlElement("form")}}。
+
+这个方法和触发提交表单按钮很类似，但有所不同：
+
+- 没有引发 {{event("submit")}} 事件。  特别是，表单的 {{domxref("GlobalEventHandlers.onsubmit", "onsubmit")}} 事件处理程序不会运行。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/submit/index.md:18]:
```suggestion
- 没有引发 {{event("submit")}} 事件。即，表单的 {{domxref("GlobalEventHandlers.onsubmit", "onsubmit")}} 事件处理程序不会运行。
```

**Comment by awxiaoxian2020** [comment]
> 



> > > > Yes, the cli only matches the prefix not the folder, I'll consider to fix or leave as it is:
> > > > https://github.com/mdn/yari/blob/bfe87cd4c0a03d5f82c2d7809ea7c449b8a7a270/markdown/cli.ts#L165-L172
> > > 
> > > 
> > > If you want to match folder (not a prefix), you could run command like this: `yarn md h2m web/api/htmlformelement/submit/ --locale zh-CN --mode dry` (adding a slash in the end)
> > 
> > 
> > OK!Thanks!
> 
> This seems not work, the slug will not end with `/` and the cli **should** match folder (not prefix), I'll working on it.

Got it, Thank you!

**Review by yin1999 — COMMENTED** [review]
I think it's better

**Pull request merged**